go/proof.go

```diff
@@ -190,7 +190,13 @@ func (p *ExistenceProof) CheckAgainstSpec(spec *ProofSpec) error {
 	if spec.MinDepth > 0 && len(p.Path) < int(spec.MinDepth) {
 		return fmt.Errorf("innerOps depth too short: %d", len(p.Path))
 	}
-	if spec.MaxDepth > 0 && len(p.Path) > int(spec.MaxDepth) {
+
+	maxDepth := spec.MaxDepth
+	if maxDepth == 0 {
+		maxDepth = 128
+	}
```

Comment:
Should we add a note to proto doc that if `max_depth` is zero then we treat as default 128?

https://github.com/cosmos/ics23/blob/master/proto/cosmos/ics23/v1/proofs.proto#L165-L166

Reply:
Sure thing :+1: